components/Table.vue

```diff
@@ -98,6 +99,11 @@
                 default:    true
             },
 
+            showEmpty: {
```

Comment:
I would add docs to this. Its kind of a confusing functionality.

Reply:
Would add the docs into the README with everything else, that way it's all in one place.